modules/admin_manual/pages/configuration/files/external_storage/dropbox.adoc

```diff
@@ -49,9 +51,11 @@ http(s)://<<Server_Address>>/index.php/settings/personal?sectionid=storage
 ....
 
 [[dropbox-install-step-three]]
-== Step Three - Create a Dropbox Share
-To create a Dropbox share, under `admin -> Settings -> Admin -> Storage`, check the "Enable external storage" checkbox, if it’s not already checked.
-Then, in the drop-down list under "External storage", click the first "Dropbox" option.
+== Create a Dropbox Share
+
+To create a Dropbox share, under menu:Admin[Settings > Storage],
```

Comment:
Is there a specific need to split sentences over multiple lines?

Reply:
see above